solana/crates/event-macros/src/lib.rs

```diff
@@ -0,0 +1,267 @@
+#![allow(missing_docs)]
+use keccak_const::Shake128;
```

Comment:
Maybe add little crate level docs - Why this create exists ?

Reply:
updated